Sources/TuistLoader/Models+ManifestMappers/Target+ManifestMapper.swift

```diff
@@ -168,7 +168,8 @@ extension TuistGraph.Target {
         let allSources = try TuistGraph.Target.sources(targetName: targetName, sources: manifest.sources?.globs.map { glob in
             let globPath = try generatorPaths.resolve(path: glob.glob).pathString
             let excluding: [String] = try glob.excluding.compactMap { try generatorPaths.resolve(path: $0).pathString }
-            return TuistGraph.SourceFileGlob(glob: globPath, excluding: excluding, compilerFlags: glob.compilerFlags)
+            let mappedCodeGen = TuistGraph.FileCodeGen.from(manifest: glob.codeGen)
```

Comment:
minor: an alternate form that can mitigate the need to deal with optionals in the `from` method would be

```swift
let mappedCodeGen = glob.codeGen.map(TuistGraph.FileCodeGen.from)
```

Reply:
Would you please bit more detailed code the above snippet returns `FileCodeGen??`

Reply:
Ah sorry, meant that would help make the `from` method be non optional:

e.g.

```swift
static func from(manifest: ProjectDescription.FileCodeGen) -> TuistGraph.FileCodeGen {
  // ...
}
```

Reply:
Fixed